source/common/tracing/zipkin/zipkin_core_types.h

```diff
@@ -171,7 +171,7 @@ class Annotation : public ZipkinBase {
   /**
    * @return true if the endpoint attribute is set, or false otherwise.
    */
-  bool isSetEndpoint() const { return endpoint_.valid(); }
+  bool isSetEndpoint() const {  if(endpoint_){return true;} return false; }
```

Comment:
This should just be `return endpoint_.has_value();`

Reply:
Agreed.